audbackend/core/api.py

```diff
@@ -92,16 +93,17 @@ def access(
 
 
 def available() -> typing.Dict[str, typing.List[Base]]:
```

Comment:
The type hint for the return value looks wrong to me, it should be:

```python
typing.Dict[str, typing.List[typing.Tuple[Base, str, str]]]
```

Reply:
I think it's correct, it looks like this because of the way we implemented `Backend.__repr__()`:

```python
backend = audbackend.Backend('host', 'repo')
repr(backend)
```
```
('audbackend.core.backend.base.Base', 'host', 'repo')
``

Reply:
Ok, I guess the confusion comes from the fact that we actually have two classes that represent an repository on a backed: `audbackend.backend.Base` which needs to be instanciated with `host` and `repo`; and we have `audbackend.Repository` which needs to instanciated with `name` (of the repository), `host` and `backend` (but name not class).

Reply:
In other packages that use `audbackend` we then take the `audbackend.Repository` class when presenting the concept of a repository to the user, e.g. `audb.Repository`.

Reply:
Yes, I think we actually introduced `audbackend.Repository` when we found such a concept was useful in `audb`. I am not a 100% sure though.